packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -154,14 +155,17 @@ export class ConnectionService {
    * @param connectionId the id of the connection for which to create a connection request
    * @returns outbound message containing connection request
    */
-  public async createRequest(connectionId: string): Promise<ConnectionProtocolMsgReturnType<ConnectionRequestMessage>> {
+  public async createRequest(
+    connectionId: string,
+    myLabel?: string
```

Comment:
I'd add the myLabel in a config like the other ones. This will make it easier to extend with more properties later on (e.g. the `imageUrl`)